src/custom/hooks/useRecentActivity.ts

```diff
@@ -127,7 +127,7 @@ export function useActivityDescriptors({ chainId, id }: { chainId?: number; id:
       // setup variables accordingly...
       const isReceiptConfirmed =
         tx.receipt?.status === TxReceiptStatus.CONFIRMED || typeof tx.receipt?.status === 'undefined'
-      isPending = !!tx?.receipt
+      isPending = !tx?.receipt
```

Comment:
This makes me think. We don't handle dropped txs do we?

Reply:
they do i believe, inside updater of transactions